### ISSUE TITLE
[BUG] Pagination List Item Link Missing href

### ISSUE THREAD

**Issue by dday9** [fiduswriter/simple-datatables#322]
**Describe the bug**
DOM elements representing the paginationListItemLink are missing an href attribute.

**To Reproduce**

I have created a [jsfiddle](https://jsfiddle.net/xkshzjfv/) here: https://jsfiddle.net/xkshzjfv that demonstrates the issue.

1. Initialize a new datatable
2. Use the following configuration for Bootstrap 5 classes
```lang-js
classes: {
    container: "table-responsive",
    dropdown: "me-auto",
    paginationList: "pagination",
    paginationListItem: "page-item",
    paginationListItemLink: "page-link",
    top: "d-flex",
}
```
3. Hover over the pagination links and notice that the "hand" cursor is missing
4. Inspect the DOM for the pagination link using your developer tools
5. Right-click on the DOM element for one of the pagination links
6. Select "Add Attribute"
7. Add the following attribute: `href="#"`

**Expected behavior**
The anchors should have a href attribute. This should probably be configurable so the user can set it using a callback, e.g.
```lang-js
pageListItemLink: page => {
  return `#/${page}`;
}
```

**Additional context**
I am setting the custom classes as pointed out in here: https://github.com/fiduswriter/simple-datatables/issues/151 to use Bootstrap for styling. I noticed that it was missing the "hand" cursor when hovering over a pagination button and adding the href solved that.

**Comment by johanneswilm** [comment]
thanks for reporting this. Fortunately, this has already been solved in the
current development version that replaces all As with BUTTONs. There will
be a release in the not too distant future.

**Comment by dday9** [comment]
That is why I absolutely love and recommend this repository, you're awesome!

Do you want me to leave this open until the release?

**Comment by johanneswilm** [comment]
@dday9 Yeah, let's keep it open for now. The reason I'm not making a release right away is that I am thinking it requires a minor and possibly a major version change and so I'm looking whether we can bundle it with other changes.

**Comment by johanneswilm** [comment]
Fixed in version 8.0.0.
